command/agent/host/windows.go

```diff
@@ -0,0 +1,58 @@
+// +build windows
+
+package host
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+)
+
+func uname() string {
+	return ""
+}
+
+func resolvConf() string {
+	return ""
+}
+
+func etcHosts() string {
+	return ""
+}
+
+func mountedPaths() (disks []string) {
+	for _, c := range "ABCDEFGHIJKLMNOPQRSTUVWXYZ" {
+		d := string(c) + ":\\"
+		_, err := os.Stat(d)
```

Comment:
Enumerating mount points on Windows is _as simple as a b c 1 2 3 you and ..._ :notes: